protocol/protocol.go

```diff
@@ -151,6 +155,8 @@ const (
 	ErrCertNotFound
 	// ErrExpired indicates that the sealed blob is no longer unsealable.
 	ErrExpired
+	// ErrCustomOpUndefined indicates that the custom opcode was invoked but no definition was provided to the server configuration.
```

Comment:
I think it's ok to reuse `ErrBadOpcode` here, and would like to avoid growing this list unless strictly required.

Reply:
That sounds good.